scripts/resolve_suggestions_urls.py

```diff
@@ -26,7 +26,7 @@ def json_path_to_pointer(json_path):
 
 
 def find_suggestion_urls(data):
-    json_path = parse("$..suggestions_url")
+    json_path = parse("$..url")
```

Comment:
Matching on `url` is too broad, if any other `url` properties were added in future this would also match them. We should use `$..suggestions.url` instead.

Reply:
Done, thanks!